src/app/interfaces/apiInterfaces.ts

```diff
@@ -41,6 +57,39 @@ export type DateTimeTimezone = DateTime;
 export function dateTimeTimezone(timestamp: string): DateTimeTimezone {
   return timestamp ? DateTime.fromISO(timestamp, { setZone: true }) : undefined;
 }
+/**
+ * Convert duration string into Duration
+ * @param seconds Duration seconds
+ */
+export function duration(seconds: number): Duration {
+  return seconds
```

Comment:
double check for me: how is `0` seconds treated here? Add a unit test or be explicit about the condition